src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -1,20 +1,25 @@
 package com.google.sps.servlets;
 
 class Constants {
+  static final String CLOSE_BRACKET = "]";
```

Comment:
Good way to make the code cleaner, especially with variables that are used consistently.